pkgs/by-name/go/goat-cli/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  fetchpatch,
+}:
+
+buildGoModule rec {
+  pname = "goat-cli";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    repo = "goat";
+    owner = "studio-b12";
+    rev = "v${version}";
+    hash = "sha256-H7ea3XOBfQ7bIX5SbxPd+fcSlMurSWXGXe+/LsqSc0A=";
+  };
+
+  vendorHash = "sha256-DtEXgGYSkWO876so6LEOkhVwDt/zrflDZdY4O2lz1mw=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/studio-b12/goat/internal/version.Version=${version}"
+    "-X github.com/studio-b12/goat/internal/version.CommitHash=${src.rev}"
+  ];
+
+  patches = [
+    ./mock-fix.patch
+  ];
+
+  # Checks currently fail because of an issue with github.com/studio-b12/goat/mocks
+  doCheck = false;
+
+  meta = {
+    description = "An integration testing tool for HTTP APIs uisng a simple script language.";
```

Comment:
```suggestion
    description = "Integration testing tool for HTTP APIs using a simple script language";
```

Reply:
Also please make sure the commit message matches the PR title. Then I can merge.